asserts/account_test.go

```diff
@@ -158,6 +158,7 @@ func (s *accountSuite) TestCheckUntrustedAuthority(c *C) {
 	otherDB := setup3rdPartySigning(c, "other", storeDB, db)
 
 	headers := ex.Headers()
+	delete(headers, "authority-id")
```

Comment:
I actually don't like the need for this change much, otoh the original test use of a junk authority is also strange :/